### PR TITLE
Bug fix for MultiClassClassification

### PR DESCRIPTION
**Pull request by bartoszpanek1**
Proposed model uses LogSoftmax activation for the last layer but during training, CrossEntropyLoss is used. In Pytorch, CrossEntropyLoss expects raw logits and it handles softmax on its own.
Therefore, current implementation is not correct because is provides log probabilities instead of raw logits for CrossEntropyLoss.

### CODE REVIEW

**Comment by wvalcke** [comment]
Yes indeed, also something i noticed. But strange enough it seems to work with CrossEntropyLoss. 
But theoretically it is wrong the way it is implemented in the video.

**Comment by wvalcke** [comment]
Another issue :

The multilabel classification is using 
loss_fn = nn.BCEWithLogitsLoss()
But the output of the network is already the sigmoid activation function output.
Or you use BCELoss(), or you adapt the network by not including the sigmoid activation function as output, but just the raw values.
